src/utils/object.js

```diff
@@ -144,12 +145,50 @@ function splitPath (path) {
   return result;
 }
 
+/**
+ * Normalize a sloppy (or dot-separated) path into a "valid" path.
```

Comment:
I'd like to just use that, but "paths" in this case have to account for both dot and slash separators.

Both possible "paths":

```
foo.bar.baz
```

```
foo/bar/baz
```